CONTRIBUTE.md

```diff
@@ -0,0 +1,45 @@
+## Setup panel
+
+1. clone the plugin to `%installationpath%/Grafana/grafana/plugins`
+2. Install dependencies `npm i`
+3. Build project `yarn build`
+
+## Setup demo env with minikube and (altered) sockshop
+
+### Minikube
+
+```
+minikube start --memory 8192 --cpus 4
+```
+
+#### Prometheus
+
+```BASH
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+helm repo update
+```
```

Comment:
i think the install line is missing "helm install [RELEASE_NAME] prometheus-community/prometheus"

Reply:
✅